rbac/internal/utils.py

```diff
@@ -36,13 +36,20 @@ def build_internal_user(request, json_rh_auth):
     user = User()
     valid_identity_types = ["Associate", "X509"]
     try:
-        if not json_rh_auth["identity"]["type"] in valid_identity_types:
+        identity_type = json_rh_auth["identity"]["type"]
```

Comment:
unrelated, I just wanted to add some logs around to help figure out why my request was failing